packages/libraries/cli/src/base-command.ts

```diff
@@ -98,7 +100,7 @@ export default abstract class BaseCommand<T extends typeof Command> extends Comm
    * @param key
    * @param args all arguments or flags
    * @param defaultValue default value
-   * @param message custom error message in case of no value
```

Comment:
Instead of a message as an argument, I standardized the message format and if a more custom message or suggested fix would be useful, then capture the error and throw the custom error.